debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java

```diff
@@ -248,13 +248,11 @@ public void changeRecord(P partition,
                 case WARN:
                     LOGGER.warn(
                             "Error while processing event at offset {}",
-                            changeRecordEmitter.getOffset().getOffset());
-                    break;
+                            changeRecordEmitter.getOffset().getOffset(), e);
```

Comment:
You should not remove the `break` statements. Also could you please try to reproduce the situation and post here from the log how the new error message looks like? Thanks a lot